### PR TITLE
Fixes dbroemme/darrens-simple-webapp#8

### PR DESCRIPTION
**Pull request by dbroemmer**
Fix the guess count issue

### CODE REVIEW

File: src/main/webapp/GuessTheNumber.jsp [hunk 1]
```diff
@@ -31,7 +31,7 @@ ${feedback}
     <label for="user_guess">Make your guess:</label>
     <input type="text" id="user_guess" name="user_guess" /><br>
     <input type="hidden" name="random_number" value="<%= randomNumber %>" />
-    <input type="hidden" name="guess_count" value="<%= guessCount + 1 %>" />
+    <input type="hidden" name="guess_count" value="<%= guessCount %>" />
```
Comment (dbroemme) [inline src/main/webapp/GuessTheNumber.jsp:34]: This should work, but lets be safe and initialize to one in this case

**Review by dbroemme — CHANGES_REQUESTED** [review]
Oen minor comment

File: src/main/webapp/GuessTheNumber.jsp (hunk 1 above)
Comment (dbroemmer) [reply src/main/webapp/GuessTheNumber.jsp:34]: Done!

**Review by dbroemmer — COMMENTED** [review]
Addressed feedback

**Review by dbroemme — APPROVED** [review]
Looks great, thanks

**Pull request merged**